release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -0,0 +1,573 @@
+====
+
+//{product-title} 4.6 is an Extended Update Support (EUS) release. More information on Red Hat OpenShift EUS is available in link:https://access.redhat.com/support/policy/updates/openshift#ocp4_phases[OpenShift Life Cycle] and link:https://access.redhat.com/support/policy/updates/openshift-eus[OpenShift EUS Overview].
+
+With the release of {product-title} 499, version 4.6 is now end of life. For more information, see the link:https://access.redhat.com/support/policy/updates/openshift[Red Hat OpenShift Container Platform Life Cycle Policy].
+
+[id="ocp-4-9-inclusive-language"]
+== Making open source more inclusive
+
+Red Hat is committed to replacing problematic language in our code, documentation, and web properties.
+
+As part of that effort, with this release the following changes are in place:
+
+* The link:https://github.com/openshift/openshift-docs[OpenShift Docs GitHub repository] `master` branch has been renamed to `main`.
+* We have begun to progressively replace the terminology of "master" with "control plane". You will notice throughout the documentation that we use both terms, with "master" in parenthesis. For example "... the control plane node (also known as the master node)". In a future release, we will update this to be "the control plane node".
+
+[id="ocp-4-9-add-on-support-status"]
+== {product-title} layered and dependant component support and compatibility
+
+The scope of support for layered and dependant components of {product-title} changes independently of the {product-title} version. To determine the current support status and compatibility for an add-on, refer to its release notes. For more information, see the link:https://access.redhat.com/support/policy/updates/openshift[Red Hat OpenShift Container Platform Life Cycle Policy].
+
+[id="ocp-4-9-new-features-and-enhancements"]
+== New features and enhancements
+
+This release adds improvements related to the following components and concepts.
+
+[id="ocp-4-9-rhcos"]
+=== {op-system-first}
+
+[id="ocp-4-9-installation-and-upgrade"]
+=== Installation and upgrade
+
+[id="ocp-4-9-web-console"]
+=== Web console
+
+[id="ocp-4-9-ibm-z"]
+=== IBM Z and LinuxONE
+
+[id="ocp-4-9-ibm-power"]
+=== IBM Power Systems
+
+[id="ocp-4-9-security"]
+=== Security and compliance
+
+[id="ocp-4-9-networking"]
+=== Networking
+
+[id="ocp-4-9-storage"]
+=== Storage
+
+[id="ocp-4-9-registry"]
+=== Registry
+
+[id="ocp-4-9-olm"]
+=== Operator lifecycle
+
+[id="ocp-4-9-osdk"]
+=== Operator development
+
+[id="ocp-4-9-builds"]
+=== Builds
+
+[id="ocp-4-9-images"]
+=== Images
+
+[id="ocp-4-9-machine-api"]
+=== Machine API
+
+[id="ocp-4-9-nodes"]
+=== Nodes
+
+[id="ocp-4-9-logging"]
+=== Red Hat OpenShift Logging
+
+In {product-title} 4.7, _Cluster Logging_ became _Red Hat OpenShift Logging_. For more information, see xref:../logging/cluster-logging-release-notes.adoc[Release notes for Red Hat OpenShift Logging].
```

Comment:
Again probably purposeful, but just checking that this logging item and the metering item below are intentionally kept.

Reply:
Yeah, leaving those for the aligned teams to make a call on :+1: